tests/client/amo/helpers.js

```diff
@@ -12,6 +12,7 @@ export const fakeAddon = {
     id: 123,
     license: { name: 'tofulicense', url: 'http://license.com/' },
     version: '2.0.0',
+    files: [],
```

Comment:
You may want to merge master into your branch just to check that this change merges cleanly. However, if github isn' complaining it's probably fine.